### PR TITLE
Windows: add Windows Store build

### PR DESCRIPTION
**Pull request by mshabunin**

### CODE REVIEW

**Comment by mshabunin** [comment]
**cc** @asmorkalov , @fengyuentau , build have failed early because we do not have UWP components installed on CI Windows machines.

**Comment by asmorkalov** [comment]
@mshabunin UWP tools are ready. Please take a look on warnings.

File: .github/workflows/OCV-PR-4.x-W10-UWP.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    paths:
+      - '.github/workflows/OCV-PR-4.x-W10-UWP.yaml'
+  workflow_call:
+
+concurrency:
+  group: OCV-PR-4.x-W10-UWP-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  EXTRA_CMAKE_OPTIONS: '-DOPENCV_DOWNLOAD_PATH=%BINARIES_CACHE% -DBUILD_opencv_gapi=OFF -DWITH_MSMF=OFF -DBUILD_TESTS=OFF -DBUILD_PERF_TESTS=OFF -DBUILD_EXAMPLES=ON -DCMAKE_BUILD_TYPE=Release'
```
Comment (asmorkalov) [inline .github/workflows/OCV-PR-4.x-W10-UWP.yaml:16]: `-DWITH_MSMF=OFF` is not correct. It should be the only back-end for UWP, if I understand correctly.
Also we can disable samples for UWP build as console apps are not relevant for the configuration.

Comment (mshabunin) [reply .github/workflows/OCV-PR-4.x-W10-UWP.yaml:16]: There is separate WinRT backend, regular MSMF can not be built due to errors, thus I've disabed it. I'll disable samples and apps build.

**Review by asmorkalov — APPROVED** [review]

**Pull request merged**